taxi/views.py

```diff
@@ -28,6 +30,24 @@ def index(request):
     return render(request, "taxi/index.html", context=context)
 
 
+def assign_driver_to_car(request, car_id, driver_id):
+    car = Car.objects.get(id=car_id)
+    driver = Driver.objects.get(id=driver_id)
```

Comment:
think of what would happen if wrong ids are passed here? What's the more appropriate method to handle this more gracefully?